rasa/core/interpreter.py

```diff
@@ -142,7 +142,7 @@ async def parse(
         if self.lazy_init and self.interpreter is None:
             self._load_interpreter()
 
-        result = self.interpreter.parse(text)
+        result = self.interpreter.parse(text, time_zone=metadata.get("time_zone"))
```

Comment:
in my opinion we should pass in the entire `metadata` as the method signature suggests it (cc @dakshvar22)

Reply:
@wochinge Shall I make this change?

Reply:
that would be great yes!

Reply:
Done.